Do not use generated postgres queries in migrations

Migrations fail on CI
```
yarn run v1.22.17
$ node scripts/migrate.js
Migration error Error: Cannot find module './generated/getTeamsByIdsQuery'
Require stack:
- /home/circleci/action/packages/server/postgres/queries/getTeamsByIds.ts
- /home/circleci/action/packages/server/postgres/utils/checkTeamEq.ts
- /home/circleci/action/packages/server/database/migrations/20220329101759-dropTeams.ts
- /home/circleci/action/node_modules/rethinkdb-ts-migrate/lib/migrate.js
- /home/circleci/action/scripts/migrate.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/circleci/action/packages/server/postgres/queries/getTeamsByIds.ts:1:196)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Module._compile (/home/circleci/action/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at newLoader (/home/circleci/action/node_modules/pirates/lib/index.js:141:7)
    at Object.newLoader [as .ts] (/home/circleci/action/node_modules/pirates/lib/index.js:141:7) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/circleci/action/packages/server/postgres/queries/getTeamsByIds.ts',
    '/home/circleci/action/packages/server/postgres/utils/checkTeamEq.ts',
    '/home/circleci/action/packages/server/database/migrations/20220329101759-dropTeams.ts',
    '/home/circleci/action/node_modules/rethinkdb-ts-migrate/lib/migrate.js',
    '/home/circleci/action/scripts/migrate.js'
  ]
}
```
https://app.circleci.com/pipelines/github/ParabolInc/parabol/6064/workflows/c37bb6f3-0bc6-4325-94ff-16920f8395f5/jobs/14863

Caused by these 2 PRs merged:
* #6239
* #6519 

We need to run the migrations first so we can generate the queries, thus we cannot use the generated queries in the migration.

## Acceptance criteria
* [ ] replace calls to generated queries with inline ones

--- a/packages/server/postgres/generatedMigrationHelpers.ts
+++ b/packages/server/postgres/generatedMigrationHelpers.ts
@@ -486,3 +486,28 @@ const updateUserTiersQueryIR: any = {
  * ```
  */
 export const updateUserTiersQuery = new PreparedQuery(updateUserTiersQueryIR)
+
+const getTeamsByIdsQueryIR: any = {
+  name: 'getTeamsByIdsQuery',
+  params: [
+    {
+      name: 'ids',
+      codeRefs: {
+        defined: {a: 39, b: 41, line: 3, col: 9},
+        used: [{a: 89, b: 91, line: 6, col: 13}]
+      },
+      transform: {type: 'array_spread'}
+    }
+  ],
+  usedParamSet: {ids: true},
+  statement: {body: 'SELECT * FROM "Team"\nWHERE id IN :ids', loc: {a: 55, b: 91, line: 5, col: 0}}
+}
+
+/**
+ * Query generated from SQL:
+ * ```
+ * SELECT * FROM "Team"
+ * WHERE id IN :ids
+ * ```
+ */
+export const getTeamsByIdsQuery = new PreparedQuery(getTeamsByIdsQueryIR)

--- a/packages/server/postgres/utils/checkTeamEq.ts
+++ b/packages/server/postgres/utils/checkTeamEq.ts
@@ -1,6 +1,7 @@
 import getRethink from '../../database/rethinkDriver'
 import Team from '../../database/types/Team'
-import getTeamsByIds from '../queries/getTeamsByIds'
+import {getTeamsByIdsQuery} from '../generatedMigrationHelpers'
+import getPg from '../getPg'
 import {checkTableEq} from './checkEqBase'
 
 const alwaysDefinedFields: (keyof Team)[] = [
@@ -12,6 +13,11 @@ const alwaysDefinedFields: (keyof Team)[] = [
   'orgId'
 ]
 const ignoredFields: (keyof Team)[] = ['updatedAt']
+
+const getTeamsByIds = async (teamIds: string[] | readonly string[]): Promise<any[]> => {
+  const teams = await getTeamsByIdsQuery.run({ids: teamIds} as any, getPg())
+  return teams
+}
 
 const maybeUndefinedFieldsDefaultValues: {[Property in keyof Partial<Team>]: unknown} = {
   jiraDimensionFields: [],

